src/main/steps/applicant1/are-you-eligible-for-residual-jurisdiction/content.ts

```diff
@@ -8,6 +8,8 @@ const en = ({ required }: CommonContent) => ({
   line1: `Are the Applicant and Respondent registered as civil partners of each other in England or Wales or,
     in the case of a same sex couple, married each other under the law of England and Wales and it would be in the
     interests of justice for the court to assume jurisdiction in this case?`,
+  yes: 'Yes',
```

Comment:
i dont think we need this as its in the common content object

Reply:
we don't, I added it because my IDE wasn't happy that the Yes and No had been added specifically for the Welsh translations. I checked and those Welsh weren't the same as the normal yes and no translation so I didn't want to change it. I can remove it if you'd still prefer though?

Reply:
![Screenshot 2022-06-21 at 11 50 21](https://user-images.githubusercontent.com/52360278/174782515-602eb800-661d-4f29-93f6-f03f65f22a73.png)

Reply:
ah okay, thats fine, best to keep it if its different translations. im happy with adding the properties to the english object or adding it to the type. either way is good.